nsv13/code/modules/overmap/starmap.dm

```diff
@@ -19,13 +19,32 @@
 	var/current_sector = 2
 	circuit = /obj/item/circuitboard/computer/ship/navigation
 
+/obj/machinery/computer/ship/navigation/can_interact(mob/user) //Override this code to allow people to use consoles when flying the ship.
+	if(locate(user) in linked?.operators)
```

Comment:
`locate()` with lists is usually used for locating types, if you're looking for a specific instance of a datum you can just use
```suggestion
	if(user in linked?.operators)
```